__tests__/nc_news_db.test.js

```diff
@@ -75,21 +77,118 @@ describe("GET /api/articles", () => {
 			.get("/api/articles")
 			.expect(200)
 			.then(({ body }) => {
-				const articlesArray = body.articles;
+				const articlesArray = body;
+				expect(articlesArray.length).not.toBe(0);
 				expect(articlesArray).toBeSortedBy("created_at", {
 					descending: true,
 				});
 			});
 	});
 });
 
+describe("GET /api/articles (sorting queries)", () => {
+	it("status: 200, returns articles sorted by created_at in descending order by default", () => {
+		return request(app)
+			.get("/api/articles")
+			.expect(200)
```

Comment:
your get request should be something like - .get("/api/articles?sorted_by=created_at&order=desc")